doc/design.md

```diff
@@ -102,37 +106,242 @@ The Reporter consumes the link status list and the specification item list and g
 ## Tracing
 
 ### Tracing Needed Coverage
-`dsn~needed_coverage_status~1`
+`dsn~tracing.needed-coverage-status~1`
 
-The Tracer component iterates over all needed Artifacts of all specification items and determines if an which coverage exists for.
+The [tracer](#tracer) component iterates over all needed artifacts of all specification items and determines if and which coverage exists for each.
 
 Covers:
 
-  * `req~needed_coverage_status~1`
+  * `req~tracing.needed-coverage-status~1`
   
 Needs: utest, impl
 
-### Backward Tracing
-`dsn~backward_coverage_status~1`
+### Outgoing Coverage Link Status
+`dsn~tracing.outgoing-coverage-link-status~1`
+
+The [linker](#linker) component determines the coverage status of the outgoing link between the provider item and the requester item.
+
+The possible results are:
 
-The Tracer component iterates over all covered IDs of all specification items and determines the backward coverage status of the link between the provider item and the requester item.
+  1. Covers:    link points to a specification item which wants this coverage
+  2. Outdated:  link points to a specification item which has a higher revision number
+  3. Predated:  link points to a specification item which has a lower revision number
+  4. Ambiguous: link points to a specification item that has duplicates
+  5. Unwanted:  coverage provider has an artifact type the provider does not want
```

Comment:
While this would be a problem in ReqM2, in OFT the artifact type is an integral part of the ID. Therefore if an item needs "dsn~foo.bar~1", the existence or non-existance of "uman~foo.bar~1" is irrelevant for said item.